library/enum.po

```diff
@@ -434,15 +436,14 @@ msgstr ""
 "attr:`_ignore_`."
 
 #: library/enum.rst:396
-#, fuzzy
 msgid ""
 "Note:  if your enumeration defines :meth:`__new__` and/or :meth:`__init__` "
 "then any value(s) given to the enum member will be passed into those "
 "methods. See `Planet`_ for an example."
 msgstr ""
-"Remarque : si l'énumération définit :meth:`__new__` ou :meth:`__init__`, "
-"alors la (ou les) valeur affectée au membre sera passée à ces méthodes. Voir "
-"l'exemple de `Planet`_."
+"Remarque : si votre énumération définit :meth:`__new__` et/ou :meth:"
```

Comment:
Pas forcément : en français, le « ou » est normalement, par convention, inclusif, contrairement à l'anglais où il faut dire « and/or » pour exprimer l'inclusif de manière non-ambigüe. Cela dit, ici le contexte peut prêter à confusion, donc ce n'est pas nécessairement une mauvaise chose.